aten/src/ATen/core/TensorAccessor.h

```diff
@@ -18,6 +18,11 @@ struct DefaultPtrTraits {
   typedef T* PtrType;
 };
 
+template <typename T>
+struct ConstPtrTraits {
```

Comment:
TODO: I realized that I don't actually need this or the `TensorBase::const_accessor` method. I can just give a const type `T` to `TensorBase::accessor<T,...>` and make it call `const_data_ptr` if the type is const